variables.tf

```
@@ -1,15 +1,57 @@
-
+variable "bucket_name" {
+  default     = null
+  description = "Name to apply to bucket (use `bucket_name` or `bucket_suffix`)"
+  type        = string
+}
 
 variable "bucket_suffix" {
-  type = string
+  default     = "default"
+  description = "Suffix to apply to the bucket (use `bucket_name` or `bucket_suffix`). When using `bucket_suffix`, the bucket name will be `[ACCOUNT_ID]-[REGION]-s3logging-[BUCKET_SUFFIX]."
```

These are TF vars, not env vars

```suggestion
  description = "Suffix to apply to the bucket (use `bucket_name` or `bucket_suffix`). When using `bucket_suffix`, the bucket name will be `[account_id]-[region]-s3logging-[bucket_suffix]."
```